### ISSUE TITLE
False positive with svelte/no-navigation-without-resolve using mailto URI schema

### ISSUE THREAD

**Issue by Jerboas86** [sveltejs/eslint-plugin-svelte#1311]
### Before You File a Bug Report Please Confirm You Have Done The Following...

- [x] I have tried restarting my IDE and the issue persists.
- [x] I have updated to the latest version of the packages.

### What version of ESLint are you using?

9.34

### What version of `eslint-plugin-svelte` are you using?

3.12.0

### What did you do?

After updating to eslint-plugin-svelte v3.12

### What did you expect to happen?

These mailto URI schema doesnt need to be resolved. No error should be detected.

### What actually happened?

The following code triggers the no-navigation-without-resolve rule as an error

```svelte
<a  href="mailto:foo@example.com">Blah</a>
```

### Link to **GitHub Repo** with Minimal Reproducible Example

I will add a repo in the following 24h

**Comment by marekdedic** [comment]
Thanks for the report, will look into it :)
